LondonBritishLibrary/orient/BLorient8821.xml

```diff
@@ -0,0 +1,297 @@
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="173ra" to="189ra"/>
+                                <title ref="LIT2509Weddas"/>
+                                <note>Arranged by the days of the week, with Monday's lesson first.</note>
+                                <msItem xml:id="ms_i1.3.1">
+                                    <locus from="173ra"/>
+                                    <title ref="LIT2509Weddas#Monday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.2">
+                                    <locus from="174va"/>
+                                    <title ref="LIT2509Weddas#Tuesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.3">
+                                    <locus from="177ra"/>
+                                    <title ref="LIT2509Weddas#Wednesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.4">
+                                    <locus from="180ra"/>
+                                    <title ref="LIT2509Weddas#Thursday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.5">
+                                    <locus from="183ra"/>
+                                    <title ref="LIT2509Weddas#Friday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.6">
+                                    <locus from="185vb"/>
+                                    <title ref="LIT2509Weddas#Saturday"/>
+                                </msItem>     
+                                <msItem xml:id="ms_i1.3.7">
+                                    <locus from="187rb"/>
+                                    <title ref="LIT2509Weddas#Sunday"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="189ra" to="196rb"/>
+                                <title ref="LIT1113Anqasa"/>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper">Oriental paper</material>
+                                    <material key="paper">European paper</material><locus target="#I #III"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+196+2</measure>
+                                    <measure unit="page" type="blank" corresp="#2v #172v">2</measure>
```

Comment:
I am not sure if corresp works here with the page references...I always used a separate locus element in these situations

Reply:
I employed the same method in the record for BLorient8824. If this is not correct, I should change it in both cases and probably also in other records.

Reply:
Yes, I have corrected it for this manuscript. I will also correct for the other BL records, that I have created up to now. I will use another branch for that.